src/Utilities/FNADllMap.cs

```diff
@@ -91,7 +91,9 @@ private static string GetPlatformName()
 		[ModuleInitializer]
 		public static void Init()
 		{
-			if (!RuntimeFeature.IsDynamicCodeCompiled)
+  			var stackTrace = new StackTrace(false);
+			var isAot = stackTrace.GetFrame(0)?.GetMethod() is null;
+			if (isAot)
```

Comment:
`StackFrame.GetMethod` will return non-null on native AOT if the method happened to be considered a target of reflection by the compiler at compile time. The api can and does return non-null with native AOT.

What problem is this code trying to solve? If setting the DllImportResolver is just unnecessary, would it be fine to just unconditionally set it?

Reply:
How we can detect none-NativeAOT if had `<PublishAot>true</PublishAot>` set in .csproj ?

Reply:
You can't. What problem is this code trying to solve?